Lobby/src/main/java/tc/oc/lobby/bukkit/gizmos/halloween/HeadlessHorseman.java

```diff
@@ -1,23 +1,25 @@
 package tc.oc.lobby.bukkit.gizmos.halloween;
 
-import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.ImmutableMap;
 import org.bukkit.Color;
 import org.bukkit.Material;
 import org.bukkit.Sound;
 import org.bukkit.entity.AbstractHorse;
 import org.bukkit.entity.EntityType;
 import org.bukkit.entity.Player;
 import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
 import org.bukkit.inventory.meta.LeatherArmorMeta;
+import tc.oc.commons.bukkit.inventory.ArmorType;
+import tc.oc.commons.bukkit.inventory.Slot;
 import tc.oc.commons.bukkit.item.ItemBuilder;
 
 public class HeadlessHorseman {
-    private final static ImmutableSet<Material> ARMOR_SET = new ImmutableSet.Builder<Material>()
-        .add(Material.PUMPKIN)
-        .add(Material.LEATHER_CHESTPLATE)
-        .add(Material.LEATHER_LEGGINGS)
-        .add(Material.LEATHER_BOOTS)
-        .build();
+    private final static ImmutableMap<Slot, ItemStack> ARMOR_MAP = ImmutableMap.of(
+        Slot.Armor.forType(ArmorType.HELMET), new ItemBuilder().material(Material.PUMPKIN).get(),
```

Comment:
Could this be a jack'o'lantern just so you don't get the annoying pumpkin overlay?